alerter.spec.js

```diff
@@ -0,0 +1,23 @@
+const { expect } = require('chai');
+let { networkAlertStub, alertInCelcius,alertFailureCount } = require('./alerter'); 
+
+describe('alertInCelcius Function', () => {
+    it('should count alert failures', () => {
+        // Stub the networkAlertStub function to return a non-ok response (500)
+        const originalNetworkAlertStub = networkAlertStub;
+        networkAlertStub = function (celcius) {
```

Comment:
Function assignment - good approach but doesn't work in this case. It had stumped me as well :)
You are assigning `networkAlertStub` over here, but it has no effect inside alerter.js.
Hint: think of `networkAlertStub` as a pointer to a function.

Reply:
I have updated the code. Hope this is what you were expecting.